### DISCUSSION TITLE
Add aux heater limits

### DISCUSSION THREAD

**Discussion by Cali1205** [emsesp/EMS-ESP32#644] [Ideas]
While working on optimizing the power consuption of my heat pump together with my PV system I thought it is a good idea to be able to control the power to the aux heater.
My idea is to use the limit setting of the aux heater.
![PXL_20220918_155339209 - Kopie](https://user-images.githubusercontent.com/11048533/192136723-bf52429c-ec92-40fa-a50b-b5d42036fbcf.jpg)

I think it would be very helpful to have these setting available.

The register seems to be 0x492.
Description | Offset
-- | --
aux heater limit with compressor active | 2
aux heater limit heating | 3
aux heater limit hot water | 4

And the values are:
Reg Value | Value
-- | --
0 | 0kW
1 | 2kW
2 | 3kW
3 | 4kW
4 | 6kW
5 | 9kW

**Comment by proddy** [comment]
add if you write to 0x492 you're able to set the kW values?

**Comment by Cali1205** [comment]
I used the mqtt command:
{"cmd":"send", "data":"0B 08 FF 02 03 92 04 04 04"}
to set the values to 6kW. 
That worked out.

**Comment by proddy** [comment]
We'll implement in https://github.com/emsesp/EMS-ESP32/issues/645

**Comment by Esquilo007** [comment]
Hello @Cali1205
Just wanted to ask if you've got some feedback regarding setting the limit of the kompresor?
I also want to optimize my PV self consumption and thus looking for the best way to do this.
How do you use it? 
Setting the Power of the heat pump according to current PV production?
Do you also have "force heating" activated if there is PV production available?
And does this work as you wished it to do?

Thank you in advance.
